### PR TITLE
Members Only Blog Project

### PR DESCRIPTION
**Pull request by ZawadiSumba66**
In this project, we built the members-only blog post in which only signed-in users can see the author of a blog otherwise they can only see the blog itself without the author

### CODE REVIEW

**Review by SmithVyne — CHANGES_REQUESTED** [review]
### Changes required 🤕 👍 
Hello @ZawadiSumba66 you have an impressive project but you need to make a few more improvements.
[REQUIRED] - You need to remove logic from your view files. Try moving them to view helpers and using partials to organize your code.

You can also reach out to me on slack, @SmithVyne if you have any other questions.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hello Team :wave:
Status: Changes Required :recycle:

Great job :clap:
I can see that you have put a lot of effort on this project. 
In addition to that, your App covers all the Behavior required but before approving this project let's cover a few details together to make your project better :blush: :arrow_double_down: . I am sure that it will not take a lot of time:

REQUIRED:

- [x] Configure your **`linter`** correctly:

As you can see, `stylelintrc.json` is missing in your project

- [x] In your readme file:
1. **Setup instructions**: Before `$ rails db:migrate` you should ask user to create databases through `$ rails db:create` then, in order to run the app, user must use the command `$ rails s`.
2. Add `screenshot` to make it professional :smiley: 

- [x] Remove comment in the user model

That's it, you can submit another code review request after making these changes
With all my respect :pray:

Rindra Josia

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.x
+      - name: Setup Rubocop
+        run: |
+          gem install --no-document rubocop:'~>0.81.0' # https://docs.rubocop.org/en/stable/installation/
+          [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ruby/.rubocop.yml
+      - name: Rubocop Report
+        run: rubocop --color
```
Comment (descholar-ceo) [inline .github/workflows/linters.yml]: [Required] This config file you used is erroneous, you should have use [this one](https://github.com/microverseinc/linters-config/blob/master/ror/.github/workflows/linters.yml) from microverse ror projects linters config files.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @ZawadiSumba66 and @SamArsanios   :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] The configuration file you used in `.github/workflows/linters.yml` doesn't seem like [this one](https://github.com/microverseinc/linters-config/blob/master/ror/.github/workflows/linters.yml) which is a mandatory file provided by microverse for all ror projects. Please fix that issue!
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by rindrajosia — APPROVED** [review]
**APPROVED** :100: 
Great work implementing the suggested changes, Congrats, your project is approved.
:1st_place_medal:  :1st_place_medal: 

You can merge it!

Good luck

Rindra Josia

Keep up the good work :muscle:

**Pull request merged**